lib_xua/src/core/endpoint0/xua_endpoint0.c

```diff
@@ -1049,7 +1070,7 @@ void XUA_Endpoint0_loop(XUD_Result_t result, USB_SetupPacket_t sp, chanend c_ep0
         if (!DFU_mode_active)
         {
 #endif
-#if (AUDIO_CLASS_FALLBACK) && (AUDIO_CLASS != 1)
+#if (XUA_AUDIO_CLASS_HS == 2) && (XUA_AUDIO_CLASS_FS == 1)
```

Comment:
This is much clearer now compared to the earlier checks using FULL_SPEED_AUDIO_2 and AUDIO_CLASS_FALLBACK